prow/plugins/lgtm/lgtm.go

```diff
@@ -33,6 +33,8 @@ import (
 const pluginName = "lgtm"
 
 var (
+	addLGTMLabelNoti    = "LGTM label has been added. Tree hash was: "
```

Comment:
Could machine-format the data in a `<details>` block as well, if we don't want to regex-munge these

Reply:
Sure, do we have an example somewhere? I have quickly grepped and couldn't find any...

Reply:
See this file: https://github.com/kubernetes/test-infra/blob/master/prow/plugins/respond.go